src/lib/components/SplitPane.svelte

```diff
@@ -42,6 +45,18 @@
       var(--header-offset, 0px) + var(--header-height)
     );
     padding-top: var(--split-pane-content-top-offset);
+    transition: padding-top var(--animation-time-normal) ease;
+
+    &.header-hidden {
+      padding-top: 0;
+      // Reset on tablet+
+      @include media.min-width(medium) {
+        padding-top: var(--split-pane-content-top-offset);
+      }
+      @include media.min-width(large) {
+        padding-top: var(--header-offset, 0px);
```

Comment:
`0` instead of `0px`? Just asking because I know sometimes the linter complains about it.